CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Aparapi Changelog
 
-## 1.4.2
+## 1.5.0
+
+* Support array in private memory. Allow to create array in private memory.
```

Comment:
Perhaps we can reword this for clarity to something like "Support inline array creation in the kernel, which is implemented in the GPU in private memory".

Reply:
done

Reply:
@shabanovd change hasnt shown up yet, but once you push that if it covers all the points we discussed ill be ready to accept and merge it.